java/code/src/com/suse/manager/webui/services/impl/runner/MgrUtilRunner.java

```diff
@@ -160,4 +160,51 @@ public static ExecResult success() {
         return call;
     }
 
+    /**
+     * {@link RunnerCall} for writing into a text file.
+     * The result of the call is a text description with how many lines were written.
+     *
+     * @param absolutePath the file path
+     * @param contents the file text contents
+     * @return the runner call
+     */
+    public static RunnerCall<String> writeTextFile(String absolutePath, String contents) {
```

Comment:
`MgrUtilRunner` is a wrapper for our custom `mgrutil` runner. Since these methods don't actually call a function from the runner I think they belong somewhere else, maybe in `SaltService`.

Reply:
I see. I got confused by the class javadoc. The methods should be definitely moved, but I think I'll put the calls to a separate class.